src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -69,6 +69,28 @@ new Sentry.Replay({
 });
 ```
 
+### Example: Capturing 500 Status Codes Only
+
+Here is an example that will only capture fetch requests that return a 500 status code. Non-fetch requests will continue to be captured as normal.
+
+```javascript
+new Sentry.Replay({
+  beforeAddRecordingEvent: (event) => {
+    // Do not capture fetch/xhr requests, unless the response code is 500
+    if (
+      event.data.tag === "performanceSpan" &&
+      (event.data.payload.op === "resource.fetch" ||
+        event.data.payload.op === "resource.xhr") &&
+      event.data.payload.data.statusCode !== 500
+    ) {
+      return null;
+    }
+
+    return event;
+  },
+});
+```
+
 Note that we also have [server-side PII scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data. It looks for certain patterns such as American social security numbers, credit cards, and private keys.
```

Comment:
```suggestion
We also have [server-side PII scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/) for this data. It looks for certain patterns such as American social security numbers, credit cards, and private keys.
```
Would be good to get rid of "Note" here since there's another sentence that starts with "Note" right below :)